testing/visual-regression/backstop-common.js

```diff
@@ -32,6 +32,21 @@ module.exports = function backstopCommon(baseUrl) {
     ];
   }
 
+  function buildSamplePageScenarios(labelPrefix, componentId) {
+    return [
+      {
+        label: `${labelPrefix} (Print)`,
+        url: storyUrlFor(componentId),
+        onReadyScript: 'onReadyEmulatePrint.js',
```

Comment:
We might want to override the viewport here for a single value that approximates an A4 page. Currently this is generating small screen versions of print styles which is a bit confusing.